### DISCUSSION TITLE
Announcing a friendly fork of dbt-docs

### DISCUSSION THREAD

**Discussion by matthieucan** [dbt-labs/dbt-docs#416] [General]
Hi,
My colleagues and I would like to announce and discuss our project to fork `dbt-docs`.
Our main reason for doing so is to be able to have a sort of staging environment, where we can build and distribute the generated `index.html`, which would contain features we intend to bring to `dbt-docs`.

Naturally, as a friendly fork, we aim to:
- Open PRs upstream (this repo) for every change we bring,
- Not diverge from upstream, except for changes that are rejected, which we'd like to maintain as a series of patches in the fork.

An example of a feature we'd like to have is https://github.com/dbt-labs/dbt-docs/pull/375. Unfortunately it didn't get a lot of traction.

What is your opinion on this approach? Do you think it's a great way for us to more efficiently work together on `dbt-docs`?

Thank you, and looking forward to working together on this topic to bring great features to `dbt-docs`!

**Comment by ismailsimsek** [comment]
@matthieucan check out [opendbt](https://github.com/memiiso/opendbt) docs. its easier to enhance. And its more polished and user friendly UI, to be used by business users.

see [documentation and the demo](https://memiiso.github.io/opendbt/)

along the [column dependencies](https://memiiso.github.io/opendbt/opendbtdocs/) you can also see the source transformations that produces the column.
